applications/registration-react/src/pages/api-registration-page/form-apiStatus/connected-form-apiStatus.test.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { createMockStore } from 'redux-test-utils';
+import shallowWithStore from '../../../../test/shallowWithStore';
+import { ConnectedFormApiStatus } from './connected-form-apiStatus';
+
+test('should render ConnectedFormApiStatus correctly', () => {
+  const testState = {};
+  const store = createMockStore(testState);
+  const wrapper = shallowWithStore(<ConnectedFormApiStatus />, store);
+  expect(wrapper).toHaveLength(1);
```

Comment:
Interessant. 
Den test gir deg dekning til connect-decorator, og du sjekker at det ikke gir feil på default gjennomgang.

Jeg anbefaler at vi klassifiserer komponenteri i roller, så at de som er klassifisert "glue" (købling andre ting til sammen) får ikke unit test (på grunn at det gjør ingenting) men i stedet, får dettet av integrasjonstetene.

Når du måler dekning (vi har fin dekningrapport), da er det lett å se hvis vi mangler integrasjonstest.

Akkurat nå har jeg ikke god ide hvordan man skulle skrive integrasjonstest til dette. Det må involvere alle komponentene som er involvert - redux og render, og connect i mellom. Vi må tenke litt.

Hva jeg ville si er, at når vi endeleig har slik integrasjonstester, skulle vi fjerne unit-teste (fra glue-component), til å få synlig integrasjontestdekning.

Samme problem kan også løses med to test suite, hver med sin eget coverage - unit test coverage og integration coverage, men hvis vi bruker samme teknologi, kan begge typer kjores i samme suite.